flytekit/clients/raw.py

```diff
@@ -1,15 +1,94 @@
 from __future__ import absolute_import
+
 from grpc import insecure_channel as _insecure_channel, secure_channel as _secure_channel, RpcError as _RpcError, \
     StatusCode as _GrpcStatusCode, ssl_channel_credentials as _ssl_channel_credentials
 from flyteidl.service import admin_pb2_grpc as _admin_service
 from flytekit.common.exceptions import user as _user_exceptions
+from flytekit.configuration.creds import (
+    CLIENT_ID as _CLIENT_ID,
+    CLIENT_CREDENTIALS_SCOPE as _SCOPE,
+)
+from flytekit.clis.sdk_in_container import basic_auth
+import logging
 import six as _six
+from flytekit.configuration import creds as _creds_config, platform as _platform_config
+
+from flytekit.clis.auth import credentials as _credentials_access
+from flytekit.clients.helpers import (
+    get_global_access_token as _get_global_access_token, set_global_access_token as _set_global_access_token
+)
+
+
+def _refresh_credentials_standard(flyte_client):
+    """
+    This function is used when the configuration value for AUTH_MODE is set to 'standard'.
+    This either fetches the existing access token or initiates the flow to request a valid access token and store it.
+    :param flyte_client: RawSynchronousFlyteClient
+    :return:
+    """
+    if not _platform_config.AUTH.get():
```

Comment:
but if not set, and admin returns a 401, this function won't do anything.  We still want to attempt to get credentials even if this is not set.  The point of this option has been coopted to mean auth will be tried regardless... that is, even if admin doesn't return a 401, auth will be attempted.

Reply:
ah sorry i didn't follow the original logic - i can bump this block back to where it was